### ISSUE TITLE
"subsciber" in the default config file

### ISSUE THREAD

**Issue by dandv** [influxdata/influxdb#7271]
I've just installed Influx v1.0.0 on Ubuntu 16.04, and the config file contains the following typo:

```
###
### [subsciber]
###
### Controls the subscriptions, which can be used to fork a copy of all data
### received by the InfluxDB host.
###

[subsciber]
  enabled = true
  http-timeout = "30s"
```

**Comment by jsternberg** [comment]
Looks like this was fixed in #7215, but it probably didn't make it into 1.0.

cc @jwilder @rossmcdonald

**Comment by jsternberg** [comment]
Fixed via #7274.
